AnywhereExamplePanel/CSXS/manifest.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ExtensionManifest Version="4.0" ExtensionBundleId="com.example.AnywherePanel" ExtensionBundleVersion="1.0.0"
+ExtensionBundleName="AnywherePanel" 
+xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<ExtensionList>
+		<Extension Id="com.example.AnywherePanel" Version="1.0" />
+	</ExtensionList>
+	<ExecutionEnvironment>
+		<HostList>
+			<Host Name="PPRO" Version="[8.0,9.9]" />
+		</HostList>
+		<CEFCommandLine>
```

Comment:
does this work? CEFCommandLine should be placed inside the DispatchInfo section. See https://wiki.corp.adobe.com/display/csxs/CEP+6+HTML+Extension+Cookbook#CEP6HTMLExtensionCookbook-CustomizeCEFcommandparameters